### ISSUE TITLE
`tns run ios --device fakeId` start random emulator

### ISSUE THREAD

**Issue by dtopuzov** [NativeScript/nativescript-cli#3625]
Run `tns run ios --device fakeId --path TestApp`.

nativescript@4.0.2:
```
Searching for devices...
Cannot find connected devices.
Simulator start failed with: No simulator image available for device identifier 'fakeId'.
To list currently connected devices and verify that the specified identifier exists, run 'tns device'.
To list available simulator images, run 'tns device <Platform> --available-devices'.
```

nativescript@next:
```
Could not find device by specified identifier 'fakeId'. To list currently connected devices and verify that the specified identifier exists, run 'tns device'.
```
...and some random simulator is started.

**Comment by Natalia-Hristova** [comment]
This is fixed in `nativescript` `rc:4.1.1-2018-06-04-11747` version and the result is the same as it was in `v.4.0.2` and no random or any simulator is started :)

**Comment by vance** [comment]
same thing... list the device, use the correct id as the param, get 'No simulator image available for device identifier'
